lib/obs_github_deployments/cli/commands/check_lock.rb

```diff
@@ -17,8 +17,9 @@ def call(**options)
           raise_error("You need to provide a repository name") unless options[:repository]
           raise_error("You need to provide a token in order to authenticate") unless options[:token]
 
-          response = client(repository: options[:repository], token: options[:token]).locked?
-          puts status_response(status: response ? "locked" : "unlocked")
+          locked = client(repository: options[:repository], token: options[:token]).locked?
+          signal_locked! if locked
+          signal_unlocked!
```

Comment:
```suggestion
          response = client(repository: options[:repository], token: options[:token]).locked?
          puts status_response(status: response ? "locked" : "unlocked")
          
          exit 1 if locked
```

Wouldn't it be easier and cleaner to leave the preious code and only add this line `exit 1 if locked`, instead of creating two more private methods?